prepare/cards/xlsum.py

```diff
@@ -0,0 +1,38 @@
+from datasets import get_dataset_config_names
+
+configs = get_dataset_config_names("GEM/xlsum")  # the languages
+# now configs is the list of all languages showing in the dataset
+
+from src.unitxt.blocks import (
+    FormTask,
+    InputOutputTemplate,
+    LoadHF,
+    TaskCard,
+    TemplatesList,
+)
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.test_utils.card import test_card
+
+langs = configs
+
+for lang in langs:
+    card = TaskCard(
+        loader=LoadHF(path="GEM/xlsum", name=lang),
+        preprocess_steps=[
+            # SplitRandomMix({"train": "train[95%]", "validation": "train[5%]", "test": "validation"}),
+            # CopyFields(field_to_field=[["answers/text", "answer"]], use_query=True),
+        ],
+        task=FormTask(
```

Comment:
In general, we should move to a summarization task that is defined outside the card.  We can setup a meeting to discuss.